legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/resources/ReviewsOnlyResource.java

```diff
@@ -0,0 +1,67 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.resources;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.finos.legend.sdlc.domain.model.project.ProjectType;
+import org.finos.legend.sdlc.domain.model.review.Review;
+import org.finos.legend.sdlc.domain.model.review.ReviewState;
+import org.finos.legend.sdlc.server.domain.api.review.ReviewApi;
+import org.finos.legend.sdlc.server.time.EndInstant;
+import org.finos.legend.sdlc.server.time.ResolvedInstant;
+import org.finos.legend.sdlc.server.time.StartInstant;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+import java.util.Set;
+
+@Path("/reviews")
+@Api("Reviews")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class ReviewsOnlyResource extends BaseResource
+{
+    private final ReviewApi reviewApi;
+
+    @Inject
+    public ReviewsOnlyResource(ReviewApi reviewApi)
+    {
+        this.reviewApi = reviewApi;
+    }
+
+
+    @GET
+    @ApiOperation(value = "Get reviews without project", notes = "Get reviews for a project. If state is provided, then only reviews with the given state are returned. Otherwise, all reviews are returned. If state is UNKNOWN, results are undefined.")
+    public List<Review> getReviews(@QueryParam("projectType") @ApiParam("If not provided or the provided. valid project types would be used") Set<ProjectType> projectType,
+                                   @QueryParam("assignedToMe") @ApiParam("show reviews assigned to me, would be set to true is not selected") Boolean  assignedToMe,
+                                   @QueryParam("state") @ApiParam("Only include reviews with the given state") ReviewState state,
+                                   @QueryParam("since") @ApiParam("This time limit is interpreted based on the chosen state: for COMMITTED state `since` means committed time, for CLOSED state, it means closed time, for all other case, it means created time") StartInstant since,
+                                   @QueryParam("until") @ApiParam("This time limit is interpreted based on the chosen state: for COMMITTED state `until` means committed time, for CLOSED state, it means closed time, for all other case, it means created time") EndInstant until,
+                                   @QueryParam("limit") @ApiParam("If not provided or the provided value is non-positive, no filtering will be applied") Integer limit)
+    {
+        return executeWithLogging(
+                (state == null) ? ("getting reviews for project type(s) "  + projectType.toString()) : ("getting reviews for project type" + projectType.toString() + " with state " + state),
```

Comment:
This will still throw a NullPointerException if projectType is null. Try this instead:
```suggestion
                (state == null) ? ("getting reviews for project type(s) "  + projectType) : ("getting reviews for project type" + projectType + " with state " + state),
```